src/components/organisms/sidebar/sidebar.tsx

```diff
@@ -173,47 +174,47 @@ export const Sidebar = () => {
 								) : null}
 							</AnimatePresence>
 						</Button>
-						{isAuthEnabled ? (
-							<>
-								<Button className="hover:bg-green-200" href="/settings" title={t("settings")}>
-									<SettingsIcon className="size-7" fill="black" />
 
+						{!isAuthEnabled ? (
+							<Popover interactionType="click">
+								<PopoverTrigger className="ml-1 flex items-center">
+									<Avatar color="black" name={user?.name} round={true} size="36" />
 									<AnimatePresence>
 										{isOpen ? (
 											<motion.span
 												animate="visible"
-												className="overflow-hidden whitespace-nowrap"
+												className="ml-2.5 overflow-hidden whitespace-nowrap text-black"
 												exit="hidden"
 												initial="hidden"
 												variants={animateVariant}
 											>
-												{t("settings")}
+												{user?.name}
 											</motion.span>
 										) : null}
 									</AnimatePresence>
-								</Button>
-
-								<Button
-									className="hover:bg-green-200"
-									onClick={() => logoutFunction()}
-									title={t("logout")}
-								>
-									<LogoutIcon className="size-7" fill="black" />
-
-									<AnimatePresence>
-										{isOpen ? (
-											<motion.span
-												animate="visible"
-												exit="hidden"
-												initial="hidden"
-												variants={animateVariant}
-											>
-												{t("logout")}
-											</motion.span>
-										) : null}
-									</AnimatePresence>
-								</Button>
-							</>
+								</PopoverTrigger>
+								<PopoverContent className="z-50 min-w-56 rounded-2xl border border-gray-950 bg-white px-3.5 py-2.5 font-averta shadow-2xl">
+									<div className="flex items-center gap-2 border-b border-b-gray-950 pb-2">
+										<Avatar color="black" name={`${user?.name}`} round={true} size="32" />
+										<span className="font-medium text-black">{user?.name}</span>
+									</div>
+									<div className="mt-1">
+										<Button
+											className="w-full rounded-md px-2.5 text-lg hover:bg-gray-250"
+											href="/settings"
+											title={t("settings")}
+										>
+											{t("settings")}
+										</Button>
+										<Button
+											className="w-full rounded-md px-2.5 text-lg hover:bg-gray-250"
+											onClick={() => logoutFunction()}
+										>
+											{t("logout")}
+										</Button>
```

Comment:
Please keep the icons for the buttons (settings & logout) like in the screenshot in the ticket

Reply:
added